PRIVACY_POLICY.md

```diff
@@ -0,0 +1,38 @@
+<!--
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+-->
+
+# Privacy Policy
+
+The Gamepad Navigator allows the user to navigate Chromium-based browsers and web pages using a game controller. It
+provides various features that allow navigation using gamepad such as web page scrolling, tab navigation, click on
+buttons and hyperlinks, et cetera. Due to the nature of browser and web page navigation, the Gamepad Navigator requires
+permission to interact with the content in the active web pages, browser windows, and tabs.
+
+## Information the Gamepad Navigator collects
+
+The Gamepad Navigator provides some web page navigation features that involve interaction with the web page's content.
+To perform certain navigation features (for example, tab navigation) correctly, the elements on the web page may be
+parsed. In addition, the Gamepad Navigator may use the custom gamepad configuration map provided by the user using the
+configuration panel.
+
```

Comment:
I like the UIO+ wording, "makes use of", and I also think you can avoid repeating yourself a bit.  Something like:

```
The Gamepad Navigator makes use of user preferences entered using the extension configuration panel.  To enable navigation features, the content of active websites is also parsed.
```